components/calendar/style/index.less

```diff
@@ -124,6 +124,15 @@
     }
   }
 
+  .@{full-calendar-prefix-cls}-small & {
+    &-value {
+      width: 20px;
+      height: 20px;
+      font-size: @font-size-sm;
+      line-height: 20px;
+    }
+  }
```

Comment:
Our Design of Calendar not include size, so this should not support.

Reply:
Got it. I will remove size props